cmd/init.go

```diff
@@ -62,11 +62,12 @@ func (c *InitCommand) Run(args []string) int {
 	s.FinalMSG = "✓ Dependencies installed\n"
 	s.Start()
 
-	err = pip.Run()
+	pipCombinedOutput, err := pip.CombinedOutput()
 
 	if err != nil {
 		s.Stop()
 		c.UI.Error(fmt.Sprintf("Error installing pip requirements: %s", err))
```

Comment:
I don't think this line is really useful (or ever was) since it just shows the exist status. Let's just combine these?

```go
c.UI.Error(fmt.Sprintf("Error installing pip requirements: \n%s", pipCombinedOutput))
```